core/src/test_fft64/automorphism_key.rs

```diff
@@ -88,26 +122,32 @@ fn test_automorphism(p0: i64, p1: i64, log_n: usize, basek: usize, k_ksk: usize,
     (0..auto_key_out.rank_in()).for_each(|col_i| {
         (0..auto_key_out.rows()).for_each(|row_i| {
             auto_key_out.get_row(&module, row_i, col_i, &mut ct_glwe_dft);
-
             ct_glwe_dft.decrypt(&module, &mut pt, &sk_auto, scratch.borrow());
-            module.vec_znx_sub_scalar_inplace(&mut pt.data, 0, row_i, &sk.data, col_i);
+
+            module.vec_znx_sub_scalar_inplace(
+                &mut pt.data,
+                0,
+                (digits_in - 1) + row_i * digits_in,
+                &sk.data,
+                col_i,
+            );
 
             let noise_have: f64 = pt.data.std(0, basek).log2();
             let noise_want: f64 = log2_std_noise_gglwe_product(
                 module.n() as f64,
-                basek,
+                basek * digits,
                 0.5,
                 0.5,
                 0f64,
                 sigma * sigma,
                 0f64,
                 rank as f64,
-                k_ksk,
-                k_ksk,
+                k_in,
+                k_apply,
             );
 
             assert!(
-                (noise_have - noise_want).abs() <= 0.1,
+                (noise_have - noise_want).abs() <= 0.5,
```

Comment:
why did they noise estimation degrade from 0.1 to 0.5?

Reply:
Because I reduced the ring degree, so there is more variance across runs